app/assets/scss/_search-page.scss

```diff
@@ -27,3 +27,10 @@
     border: 1px solid $border-colour;
   }
 }
+
+.search-result-excerpt {
+  em {
```

Comment:
The toolkit [already has SASS for `.search-result-highlighted-text`](http://alphagov.github.io/digitalmarketplace-frontend-toolkit/search-result.html).